### DISCUSSION TITLE
[Feature Request] WeatherAPI

### DISCUSSION THREAD

**Discussion by Tataukee** [daledavies/jump#63] [Feature Requests]
Can you add weatherapi ? The openweather has never been working for me ?

**Comment by daledavies** [comment]
Howdy, thanks for reaching out. 

Is this weatherapi.com? 

If so it looks like they require a monthly subscription to access the API, I can only support free services unfortunately. Yes there is a free trial but beyond that I wouldn't be able to offer any support should there be issues.

I'm interested to know what is not working for you with open weather map, is it something in jump that isn't working correctly?

**Reply by Tataukee** [reply]
Hi, 
My Jump dashboard was just the background with the top info visible and nothing else when I include my OpenWeather API Key . However when i leave the "OWMAPIKEY" environnement blank in the docker compose and save it , Jump will work after refreshing the container .
I believe the issue is related with the "OWAPI Key" but mine is active .

![03Oct2220-07](https://user-images.githubusercontent.com/26170999/193648201-13010380-a5e7-4808-be4e-5f81f3e504c6.png)

**Reply by Tataukee** [reply]
Yes it's weatherapi.com . I currently use their api for some other usage and it works .
They have free plan .

![03Oct2220-16](https://user-images.githubusercontent.com/26170999/193649705-04746304-55e4-49be-9eb6-c49929e5de97.png)

**Comment by daledavies** [comment]
I've created an issue from this discussion requesting some additional details that will help me debug this... https://github.com/daledavies/jump/issues/64
